src/modules/wrapper/wrapper.component.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { Observable } from 'rxjs/Observable';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ActivatedRoute, Router } from '@angular/router';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+
+import { expect } from '@blackbaud/skyux-builder/runtime/testing/browser';
+import { SkyAppWindowRef } from '@blackbaud/skyux-builder/runtime';
+
+import { StacheWrapperComponent } from './wrapper.component';
+import { StacheLayoutComponent } from '../layout/layout.component';
+import { StacheWrapperTestComponent } from './fixtures/wrapper.component.fixture';
+import { StacheTitleService } from './title.service';
+
+describe('Stache Wrapper Component', () => {
+  let component: StacheWrapperComponent;
+  let fixture: ComponentFixture<StacheWrapperComponent>;
+  let testFixture: ComponentFixture<StacheWrapperTestComponent>;
+  let mockActivatedRoute: any;
+  let mockRouter: any;
+  let mockTitleService: any;
+  let mockWindowService: any;
+
+  class MockRouter {};
+
+  class MockActivatedRoute {
+    public fragment = Observable.of('test-route');
+  };
+
+  class MockTitleService {
+    public setTitle = jasmine.createSpy('setTitle');
+  };
+
+  class MockWindowService {
+    public nativeWindow = {
+      document: {
+        getElementById: jasmine.createSpy('getElementById').and.returnValue({
+            scrollIntoView: jasmine.createSpy('scrollIntoView');
+          });
+      },
+      setTimeout: jasmine.createSpy('setImmediate').and.callFake(function(callback: any) {
+        return callback();
+      })
+    };
+  };
+
+  beforeEach(() => {
+    mockActivatedRoute = new MockActivatedRoute();
+    mockRouter = new MockRouter();
+    mockTitleService = new MockTitleService();
+    mockWindowService = new MockWindowService();
+
+    TestBed.configureTestingModule({
+      declarations: [
+        StacheLayoutComponent,
+        StacheWrapperComponent,
+        StacheWrapperTestComponent
+      ],
+      providers: [
+        { provide: ActivatedRoute, useValue: mockActivatedRoute },
+        { provide: Router, useValue: mockRouter },
```

Comment:
Both `mockActivatedRoute` and `mockRouter` are not being used in the tests, so you can instead inject the class itself into the providers:

```
{ provide: ActivatedRoute, useClass: MockActivatedRoute }
```